doc/source/user_guide/groupby.rst

```diff
@@ -199,6 +199,27 @@ For example, the groups created by ``groupby()`` below are in the order they app
    df3.groupby(['X']).get_group('B')
 
 
+.. _groupby.dropna:
+
+GroupBy dropna
+^^^^^^^^^^^^^^
+
+By default ``NA`` values are excluded from group keys during the ``groupby`` operation. However,
+in case you want to include ``NA`` values in group keys, you could pass ``dropna=False`` to achieve it.
+
+.. ipython:: python
+
+    df_list = [[1, 2, 3], [1, None, 4], [2, 1, 3], [1, 2, 2]]
+    df_dropna = pd.DataFrame(df_list, columns=["a", "b", "c"])
+
+    df_dropna.groupby(by=["b"], dropna=False).sum()
```

Comment:
reorg these to be

```
# say this is the default
df_dropna.groupby(by=["b"], dropna=True).sum()

# to allow NaN keys
```

but use full sentences and such

Reply:
changed!